### PR TITLE
New Highlight Command

### PR DESCRIPTION
**Pull request by LiveFireExercise**
Utilising a very obscure Roblox object to do so.

### CODE REVIEW

**Comment by h0nzzz** [comment]
`player` doesnt seem to be defined anywhere (or i might just be dumb)

**Comment by DimensionalFusion** [comment]
No it’s me

**Comment by P3tray** [comment]
> No it’s me

fix it then dummy

File: MainModule/Server/Commands/Moderators.lua [hunk 1]
```diff
@@ -6866,5 +6866,22 @@ return function(Vargs, env)
...
+			AdminLevel = "Moderators";
+			Function = function(plr: Player, args: {string})
+				assert(args[1], "Missing target player")
+				local Highlight = Instance.new("Highlight")
+				Highlight.Name = "Adonis_Highlight"
+				Highlight.OutlineTransparency = 0.5
+				Highlight.FillTransparency = 0.25
+				Highlight.FillColor = Color3.fromRGB(110,209,255)
+				Highlight.DepthMode = Enum.HighlightDepthMode.Occluded
+				Highlight.Parent = player
```
Comment (P3tray) [inline MainModule/Server/Commands/Moderators.lua:6883]: Change `Highlight.Parent = player` to `Highlight.Parent = getplayer()` or whatever basically your an idiot.

**Review by P3tray — CHANGES_REQUESTED** [review]

**Comment by joritochip** [comment]
Closing due to no activity to fix issue from pull request author, you can reopen it but if you do also make sure to include a command to un-highlight someone

**Comment by P3tray** [comment]
@LiveFireExercise your lazy

**Pull request closed without merging**